components/org.wso2.carbon.identity.application.authenticator.samlsso/src/main/java/org/wso2/carbon/identity/application/authenticator/samlsso/util/SSOConstants.java

```diff
@@ -57,6 +61,10 @@ private StatusCodes() {
 
         public static final String IDENTITY_PROVIDER_ERROR = "urn:oasis:names:tc:SAML:2.0:status:Responder";
         public static final String NO_PASSIVE = "urn:oasis:names:tc:SAML:2.0:status:NoPassive";
+        public static final String SUCCESS_CODE = "urn:oasis:names:tc:SAML:2.0:status:Success";
+        public static final String REQUESTOR_ERROR = "urn:oasis:names:tc:SAML:2.0:status:Requester";
+        public static final String VERSION_MISMATCH = "urn:oasis:names:tc:SAML:2.0:status:VersionMismatch";
```

Comment:
Any specific reason not to use the constants in https://github.com/apigee/java-opensaml2/blob/master/src/main/java/org/opensaml/saml2/core/StatusCode.java which is in OpenSAML2?